Assets/Scripts/Spike.cs

```diff
@@ -5,11 +5,48 @@
 
 public class Spike : MonoBehaviour
 {
+    private CloseOrOpenCircle m_HoleTransition;
+
+    private static bool S_Collided = false; //prevents multiple collision events spamming
+
+    private void Start()
+    {
+        S_Collided = false;
+        if (GameObject.Find("HoleTransition") == null)
+        {
+            Debug.Log("The LevelTransitioner Prefab should be in the level. Put at the above the other UI in the hierachy. Ask Sach for more help. Spikes will still work without it :D");
```

Comment:
"put at the above the other..." hahaha
but if the LevelTransitioner should be in the scene from now on, maybe add it to the DaddyObject